Importing createShallow from @material-ui/core/test-utils broke in v4.0.0-rc0

<!--- Provide a general summary of the issue in the Title above -->
I use `createShallow` in my tests. Since upgrading to v4.0.0-rc0 I'm getting the following error:
```
[Error] Runtime error: Error: Cannot find module 'test/utils/RenderMode' from 'createRender.js' 
```
The import at fault is:
https://github.com/mui-org/material-ui/blob/f32d82e58ef702c520891e2b32d7a6f5dd72e494/packages/material-ui/src/test-utils/createRender.js#L3

I can't figure out where that import is supposed to come from. Am I missing something?
<!--
    Thank you very much for contributing to Material-UI by creating an issue! ❤️
    To avoid duplicate issues we ask you to check off the following list.
-->

<!-- Checked checkbox should look like this: [x] -->
- [x] This is not a v0.x issue. <!-- (v0.x is no longer maintained) -->
- [x] I have searched the [issues](https://github.com/mui-org/material-ui/issues) of this repository and believe that this is not a duplicate.

## Expected Behavior 🤔
<!---
    Describe what should happen.
-->
The following should work:
import { createShallow } from '@material-ui/core/test-utils';
## Current Behavior 😯
<!---
    Describe what happens instead of the expected behavior.
-->
The following error:
```
[Error] Runtime error: Error: Cannot find module 'test/utils/RenderMode' from 'createRender.js' 
```
## Steps to Reproduce 🕹
<!---
    Provide a link to a live example (you can use codesandbox.io) and an unambiguous set of steps to reproduce this bug.
    Include code to reproduce, if relevant (which it most likely is).

    This codesandbox.io template _may_ be a good starting point:
    https://codesandbox.io/s/github/mui-org/material-ui/tree/next/examples/create-react-app

    If you're using typescript a better starting point would be
    https://codesandbox.io/s/github/mui-org/material-ui/tree/next/examples/create-react-app-with-typescript

    If YOU DO NOT take time to provide a codesandbox.io reproduction, should the COMMUNITY take time to help you?

-->
1. Import { createShallow } from `@material-ui/core/test-utils`.
2.
3.
4.

## Context 🔦
<!---
    What are you trying to accomplish? How has this issue affected you?
    Providing context helps us come up with a solution that is most useful in the real world.
-->

## Your Environment 🌎
<!---
    Include as many relevant details about the environment with which you experienced the bug.
    If you encounter issues with typescript please include version and tsconfig.
-->
I use Jest as my test runner.
| Tech         | Version |
|--------------|---------|
| Material-UI  | v4.0.0-rc1  |
| React        | 16.8.6 |
| Jest         |  23.6.0 |

Yep me too, tests broke with this error message 

    ● Test suite failed to run

    Cannot find module 'test/utils/RenderMode' from 'createRender.js'

    However, Jest was able to find:
    	'./createRender.d.ts'
    	'./createRender.js'


    You might want to include a file extension in your import, or update your 'moduleFileExtensions', which is currently ['js', 'json', 'jsx', 'ts', 'tsx', 'node'].

    See https://jestjs.io/docs/en/configuration#modulefileextensions-array-string

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:230:17)
      at Object.<anonymous> (node_modules/@material-ui/core/test-utils/createRender.js:18:19)

--- a/packages/material-ui/src/test-utils/RenderMode.js
+++ b/packages/material-ui/src/test-utils/RenderMode.js
@@ -3,6 +3,9 @@ import * as PropTypes from 'prop-types';
 
 const Context = React.createContext();
 
+/**
+ * @ignore - internal component.
+ */
 export function RenderContext({ children }) {
   return <Context.Provider value="render">{children}</Context.Provider>;
 }

--- a/packages/material-ui/src/test-utils/createRender.js
+++ b/packages/material-ui/src/test-utils/createRender.js
@@ -1,6 +1,6 @@
 import { render as enzymeRender } from 'enzyme';
 import React from 'react';
-import { RenderContext } from 'test/utils/RenderMode';
+import { RenderContext } from './RenderMode';
 
 // Generate a render to string function.
 export default function createRender(options1 = {}) {

--- a/test/utils/init.js
+++ b/test/utils/init.js
@@ -2,7 +2,7 @@ import React from 'react';
 import enzyme from 'enzyme/build/index';
 import Adapter from 'enzyme-adapter-react-16';
 import consoleError from './consoleError';
-import { useIsSsr } from './RenderMode';
+import { useIsSsr } from '@material-ui/core/test-utils/RenderMode';
 
 consoleError();
 

